projects/kubernetes-sigs/cluster-api/patches/0001-Adding-capi-support-for-Bottlerocket.patch

```diff
@@ -1,7 +1,7 @@
-From 1a08c390a95fad1a448f41f4cb2d5187daa16ac3 Mon Sep 17 00:00:00 2001
+From 858e8c1e1389a16ef9282559fee0317c25604597 Mon Sep 17 00:00:00 2001
```

Comment:
Hmm, how did you apply the patches? These shouldn't change in the existing patches

Reply:
My Slack message from Abhay:
> I don't think it matters if you change that commit coz it's only going to get applied in the CI environment anyway, and it'll apply cleanly with the old commit ID too in my experience